src/web/auth.rs

```diff
@@ -116,6 +116,7 @@ impl HawkPayload {
         // Comment the following to disable auth
         verify_hmac(payload, &secrets.signing_secret, signature)?;
 
+        dbg!(std::str::from_utf8(payload).unwrap());
```

Comment:
Couple things:

- we don't want to remove user_id from our primary key, but it should be switched from the `legacy_id` u64 for mysql to the `fxa_uid` String from the hawk header (you can see it in this dbg! statement)
- HawkIdentifier already has a field for it: `fxa_id`, let's rename it to `fxa_uid`
- And the rest of this patch pretty much gets the second part: the addition of `fxa_kid` to the primary key. The "kid" column name sounds good but its value should come into the db methods as `params.user_id.fxa_kid` to be clear and distinguished from `params.user_id.fxa_uid`
- HawkParameter (which passes through the hawk json) also needs these new fields added

So our primary key will generally be`(user_id, kid, collection)`.

Reply:
ok, added a DDL file containing the new spanner schema to this PR.